scss/widgets/material/treeView/_index.scss

```diff
@@ -18,7 +18,16 @@ $material-treeview-container-padding:
   $material-treeview-border-padding
   $material-treeview-container-with-border-offset;
 
-$material-treeview-search-editor-height: 33px;
+$material-treeview-search-editor-height: null !default;
+
+@if $size == "default" {
+  $material-treeview-search-editor-height: 48px;
```

Comment:
i think we should move it to "sizes" file

Reply:
https://github.com/DevExpress/DevExtreme/pull/23649/commits/99f3c58b1d0e9057a7f93892f817ef25839b119c